scvi/model/base/_archesmixin.py

```diff
@@ -41,7 +41,7 @@ def load_query_data(
         ----------
         adata
             AnnData organized in the same way as data used to train model.
-            It is not necessary to run :func:`~scvi.data.setup_anndata`,
+            It is not necessary to run setup_anndata,
```

Comment:
```suggestion
            It is not necessary to run `setup_anndata`,
```

This happens in several places, the ` just make it format in italics on docs

Reply:
do we want to use code formatting instead (double back ticks)